docs/contribute/processes/release.md

```diff
@@ -18,6 +18,10 @@ Please make sure before creating the artifacts that **all unit and stf tests are
 * Update Site
 * Drop-in
 
+#### Prerequisite
+Make sure to increase the version number of the `saros.eclipse` **and** `saros.core` osgi bundle.
```

Comment:
As a suggestion:

```suggestion
If there were any changes to the Saros core since the last release, make sure to also increase the version number of the `saros.core` osgi bundle.
```